applications/luci-app-email/htdocs/luci-static/resources/view/email/emailrelay.js

```diff
@@ -0,0 +1,306 @@
+		o.default = '<default>';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('server_tls', '1');
+
+		o = s.taboption('smtp_server', form.Value, 'spool_dir', _('Mail storage directory'),
+			_(
+				'The directory used for holding received mail messages.<br/>' +
+				'<b>Note:</b> The <code>/var/</code> is a small in-memory folder and you\'ll lose mail in reboot.<br/>' +
+				'Instead, use a mounted disk with enough of space.'
+			)
+		);
+		o.datatype = 'directory';
+		o.default = '/var/spool/emailrelay';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+
+
+		o = s.taboption('smtp_client', form.Value, 'smarthost', _('Smarthost'),
+			_('The SMTP server to forward emails')
+		);
+		o.datatype = 'host';
+		o.optional = false;
+		o.rmempty = false;
+		o.depends('mode', 'proxy');
+
+		o = s.taboption('smtp_client', form.Value, 'client_auth', _('Client authorization file'),
+			_(
+				'A file that contains credentials for SMTP smarthost client.<br/>' +
+				'See <a href="%s#reference_md_Authentication" target="_blank" rel="noreferrer">Authentication</a>'
+			).format(docsRef)
+		);
+		o.datatype = 'file';
+		o.default = '/etc/emailrelay.auth';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('mode', 'proxy');
+
+		o = s.taboption('smtp_client', form.Flag, 'client_tls', _('Enable SMTP client TLS'),
+			_(
+				'Use TLS encryption for SMTP proxy client.<br/>' +
+				'See example for <a href="%s#userguide_md_Google_mail" target="_blank" rel="noreferrer">Gmail</a>'
+			).format(docsRef)
+		);
+		o.default = '0';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('mode', 'proxy');
+
+		o = s.taboption('smtp_client', form.Value, 'client_tls_key', _('TLS private key'),
+			_('Path to TLS private key. ')
+		);
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+		o.depends('client_tls', '1');
+
+		o = s.taboption('smtp_client', form.Value, 'client_tls_certificate', _('TLS certificate'),
+			_('Path to TLS cert. ')
+		);
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('client_tls', '1');
+
+		o = s.taboption('smtp_client', form.Value, 'client_tls_verify', _('CA certificate'),
+			_(
+				'Verify an SMTP server\'s certificate against any of the trusted ' +
+				'CA certificates in the specified file or directory.<br/>' +
+				'Use <code>&lt;default&gt;</code> to use the system trusted CAs'
+			)
+		);
+		o.datatype = 'or(file, directory, "<default>")';
+		o.default = '<default>';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('client_tls', '1');
+
+
+		o = s.taboption('pop_server', form.Flag, 'pop', _('Enable POP3'),
+			_(
+				'The POP3 used to fetch a mail.<br/>' +
+				'See <a href="%s#userguide_md_Running_as_a_POP_server" target="_blank" rel="noreferrer">Running as a POP server</a>'
```

Comment:
We should also move the the HTML link tags into the format so the translater doe not need to deal with HTML tags.
So we have:
```
_('The POP3 used to fetch a mail.) + <br/>' +
_('See <a "%s">Running as a POP server</a>')
.format('href="https://emailrelay.sourceforge.net/#userguide_md_Running_as_a_POP_server" target="_blank" rel="noreferrer"')
```

Reply:
fixed